build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euxo pipefail
+
+NO_CACHE=""
+export DOCKER_BUILDKIT=0
+BASE_CONTAINER="jupyter/scipy-notebook:c7fb6660d096"
+PYTHON2_IMG="ypetit/test:python-2.7"
```

Comment:
:thinking:

Reply:
Tant que la MR gitlab sur le projet docker-image n'est pas passée je ne peux pas mettre un tag officiel saagie ....

Reply:
images have been changed to saagie's ones (from branch) will be updated to saagie's one from master asap